back-end/app.js

```diff
@@ -363,8 +385,17 @@ app.get('/shows-trakt/:id', (req, res, next) => {
           } //send movie info and, if available, poster info
           res.json(response_final)
         })
-        .catch((err) => {
-          next(err)
+         .catch((err) => {
+          if (err.status != 200 && err.status != 304) {
```

Comment:
I believe from my previous work with the error objects, if you want to access the status code, it'll be `err.response.status` instead of `err.status`, so I think it'd be good to change this.